.github/workflows/ci.yml

```diff
@@ -155,7 +155,7 @@ jobs:
 
   static-checks:
     timeout-minutes: 30
-    name: "Static checks: no pylint"
+    name: "Static checks: except pylint"
```

Comment:
I think it's better. But maybe we should rename the other step to simply `Pylint` and name this one as `Static checks` in this way there will be no wording problem I think.

Reply:
Just one watch out: https://github.com/apache/airflow/blob/master/.github/workflows/build-images-workflow-run.yml#L102. I think "Pylint" and "Static checks" is good but we have to update the cancelling workflow to make sure the new name is matched by the regexp :). No harm if it's not - but if it matches, then a failure in this step will trigger cancelling the whole workflow when someone starts another PR.

Reply:
Updated